FormlyMapper/FormlyField.php

```diff
@@ -35,9 +35,15 @@ public function getFormlyFieldConfiguration()
      */
     protected function buildConfiguration()
     {
+        $this->formlyFieldConfiguration = [];
```

Comment:
When i was doing functional testing, I made a `foreach` to retrieve all the forms, so the `formlyFieldConfiguration` was keeping the last value of other field and i need to retrieve just the actual configuration for the actual field, thats the reason to initialize here.